templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete driver?</h1>
+  <p class="alert alert-warning">
+    Are you sure you want to delete this driver? This action cannot be undone.
+  </p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger">
+    <a href="{{ request.META.HTTP_REFERER }}" class="btn btn-secondary">Cancel</a>
```

Comment:
Using `request.META.HTTP_REFERER` for the cancel link might not always return the expected previous page. Consider using a more reliable method, such as passing the previous URL in the context or using a named URL pattern.